### PR TITLE
chore: update project docs

### PR DESCRIPTION
**Pull request by jleach**
# Summary of Changes

Updated docs to reflect we are part of the OWF now and not Hyperledger.

# Related Issues

n/a

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,47 +2,55 @@
...
 
-Designed with React Native, Bifold offers cross-platform capabilities, supporting a wide range of use-cases and devices. Bifold is built on the Aries Framework Javascript (AFJ), providing a solid foundation for its architecture.
+**Key Features and Benefits:**
 
-Our aim is to cultivate a collaborative environment, bringing together organizations and individuals to prevent duplicate efforts and to promote shared solutions. By doing so, we hope to accelerate the adoption of VC technology across various sectors.
+- **Unified Digital Identity Management:** Emphasizing security and user-friendliness, Aries Bifold excels in consolidating and managing digital identities across various standards like AnonCreds and W3C VC Data Model. This capability positions Bifold as a pivotal resource for secure and private handling of digital identities, accessible to all.
```
Comment (bryce-mcmath) [inline README.md]: We should remove the "Aries" here, no?

File: README.md [hunk 2]
```diff
@@ -2,47 +2,55 @@
...
+- **Seamless Multi-Platform Use:** Thanks to its React Native architecture, Bifold delivers a smooth experience on any device, enabling users to manage their digital identities whether they are using a phone or a tablet. This cross-platform flexibility means that developers can create applications once and deploy them on both iOS and Android, ensuring a consistent and accessible user experience.
 
-Join us as we work towards a future where verifiable credentials are easily accessible and widely adopted, enhancing trust and security in the digital world.
+- **Community-Driven Development:** Aries Bifold is more than a tool; it's a community initiative aimed at fostering collaboration and sharing innovations. By bringing together diverse groups, from organizations to individuals, Bifold encourages the pooling of resources and knowledge to facilitate the broader adoption and understanding of verifiable credentials.
```
Comment (bryce-mcmath) [inline README.md]: Here as well

**Review by bryce-mcmath — CHANGES_REQUESTED** [review]

**Comment by bryce-mcmath** [comment]
Can we get this one fixed up and merged?

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -2,76 +2,32 @@
...
 ## Attribution
 
-This Code of Conduct is adapted from the code of conduct for the [Hyperledger
-Aries Cloud Agent -
-Python](https://github.com/hyperledger/aries-cloudagent-python) project, which
-in turn is adapted from the [Contributor Covenant][homepage], version 1.4,
-available at [http://contributor-covenant.org/version/1/4][version]
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], [version 1.4](version).
 
 [homepage]: http://contributor-covenant.org
```
Comment (wadeking98) [inline CODE_OF_CONDUCT.md:31]: I think maybe these link definitions need to be at the top? the links aren't working for me

File: README.md [hunk 4]
```diff
@@ -2,47 +2,55 @@
...
-1. [Developers Guide](./DEVELOPER.md)
+1. [Developer's Guide](developers-guide)
 
 # Success Stories
 
 The organizations or jurisdictions listed below have either deployed a project based on Bifold, are currently assessing the technology, or are actively developing a solution. If you want your organization to be included in this list, please submit a Pull Request against this README.md.
 
 - [BC Wallet](https://apps.apple.com/us/app/bc-wallet/id1587380443)
   The BC Wallet is a digital wallet app developed by the Government of British Columbia.
+
```
Comment (wadeking98) [inline README.md:53]: same issue here

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (jleach) [reply CODE_OF_CONDUCT.md:31]: They should work once in the web.

**Comment by bryce-mcmath** [comment]
Still seems to have conflicts

**Review by bryce-mcmath — APPROVED** [review]

**Pull request merged**